src/App.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div id="app">
-    <tx-preloader v-if="isLoading" :progress="progress" />
+    <preloader v-if="isLoading" :progress="progress" />
```

Comment:
Component name should always be multi-word. This is what Vue team recommended https://v2.vuejs.org/v2/style-guide/?redirect=true#Multi-word-component-names-essential

Reply:
If you want to remove the Tx prefix, please make sure your component using multi-word naming convention

Reply:
I decided to add V to my components name as a prefix as it said https://v2.vuejs.org/v2/style-guide/#Base-component-names-strongly-recommended

Reply:
ok